Formula/mask.rb

```diff
@@ -15,7 +15,7 @@ class Mask < Formula
   depends_on "rust" => :build
 
   def install
-    system "cargo", "install", *std_cargo_args
+    system "cargo", "install", "--locked", "--root", prefix, "--path", "./mask"
```

Comment:
```suggestion
    cd "mask" do
      system "cargo", "install", *std_cargo_args
    end
```
Think this should work.

Reply:
Thanks! 🤞